src/components/PriceLine.jsx

```diff
@@ -41,4 +42,36 @@ class PriceLine extends Component {
     }
 }
 
+PriceLine.propTypes = {
+    top             : PropTypes.number,
+    className       : PropTypes.string,
+    draggable       : PropTypes.bool,
+    isDragging      : PropTypes.bool,
+    priceDisplay    : PropTypes.number,
+    setDragLine     : PropTypes.any,
+    visible         : PropTypes.bool,
+    zIndex          : PropTypes.number,
+    uncentered      : PropTypes.bool,
+    offScreen       : PropTypes.bool,
+    lineStyle       : PropTypes.string,
+    color           : PropTypes.string,
+    foregroundColor : PropTypes.string,
+};
+
+PriceLine.defaultProps = {
```

Comment:
@shayanbinary totally, there isn't any need for the default values for this component.